ui.frontend/src/utils.js

```diff
@@ -197,7 +197,7 @@ export default class Utils {
                 const params = Object.fromEntries(urlSearchParams.entries());
                 let _prefillData = {};
                 if (_formJson) {
-                    _prefillData = await HTTPAPILayer.getPrefillData(_formJson.id, params) || {};
+                    _prefillData = await HTTPAPILayer.getPrefillData(btoa(_path), params) || {};
```

Comment:
this is not correct. we should not decode the id at client.

Reply:
currently there's no other way to provide the encoded data for Sites page, since id will be different now